docs/central-users.rst

```diff
@@ -226,26 +227,32 @@ To manage App Users, navigate to the project whose App Users you wish to manage,
 Creating an App User
 ~~~~~~~~~~~~~~~~~~~~
 
-To create a new App User, click on the :guilabel:`Create app user` button on the right side of the App Users listing page. You will see a popup that looks like this:
+To create a new App User, click on the :guilabel:`Create app user` button on the right side of the App Users listing page.
+
+First, you will need to provide a Display Name for the new user. This name will appear in data tables, exports, and system logs.
+
+Ideally, every individual involved in data collection gets their own App User account. This makes it easiest to track who did what and to ensure that only authorized individuals are submitting data. In that case, the App User Display Names are the names of the individuals. However, this is not always practical, especially if you have a lot of data collectors. If you have a lot of data collectors and want to give them each their own App User account, consider using a script such as `this pyodk-based one <https://getodk.github.io/pyodk/examples/app_user_provisioner/app_user_provisioner/>`_ to create App Users and manage form assignments.
```

Comment:
People won't know what pyodk is, so s/pyODK/Python/g/. Also mention bulk? And hint at technical requirement.

> If you have a lot of data collectors and want to give them each their own App User account, consider using a script such as `this Python one <https://getodk.github.io/pyodk/examples/app_user_provisioner/app_user_provisioner/>`_ to bulk create App Users and manage form assignments via the API.